panel/_package-lock.json

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
Why check this in if regular old package-lock.json is now checked in?

Reply:
actually I haven't succeed to use jszip and vtk in typescript for the moment
jszip I have an error with readable-stream
and vtkjs on importation.

Reply:
I'm actually currently a -1 on bundling vtk.js with panel.js. Once bokeh allows multiple bundles in an extension we can add a panel-vtk.js bundle but for now it's too large to be included by default.

Reply:
this is the error I get when trying to add jszip and @types/jszip to node modules:
`Error: can't resolve 'stream' from 'C:\Users\XA235208\Workspace\PythonWorkspace\modules\panel\panel\node_modules\readable-stream\readable.js'`
may be @mattpap  has an idea to solve it